server/middleware/permissionsMiddleware.ts

```diff
@@ -0,0 +1,40 @@
+import { RequestHandler } from 'express'
+import { services } from '../services'
+import logger from '../../logger'
+import AuthorisedRoles from '../authentication/authorisedRoles'
+import Permissions from '../authentication/permissions'
+
+export default function populateUserPermissions(): RequestHandler {
+  const { keyworkerApiService } = services()
+
+  return async (req, res, next) => {
+    try {
+      const userIsKeyworker =
```

Comment:
This is somewhat misleadingly named. Having the KEYWORKER_MONITOR role doesn't make them a key worker so I would change this to userHasViewPermission or simply add to the permissions collection straight away